Loop/Managers/NotificationManager.swift

```diff
@@ -130,6 +153,9 @@ extension NotificationManager {
 
         notification.body = body
         notification.sound = .default
```

Comment:
Some had these interruptionLevel and others did not - making consistent.

Reply:
I'm not sure a failed remote command is of any urgency to the loop user; it's important to the caregiver.

Reply:
Good point - I've removed.